Dockerfile.rollup.in

```diff
@@ -0,0 +1,10 @@
+FROM registry.fedoraproject.org/fedora:28
+RUN yum -y install ostree && yum clean all
+# These variables are templated 
+ADD rollup.sh /root
+RUN env OSTREE_REPO_URL=@OSTREE_REPO_URL@ /root/rollup.sh && rm -f /root/rollup.sh
+# Keep this in sync with Dockerfile
+COPY nginx.conf /etc/nginx/nginx.conf
+COPY index.html subdomain.css /srv/tree/repo/
+EXPOSE 8080
+CMD ["nginx", "-c", "/etc/nginx/nginx.conf"]
```

Comment:
do we need to install nginx on fedora img, is it included by default?

Reply:
Thanks, yeah I hadn't tested the http functionality.   Fixed :arrow_down: